src/App.js

```diff
@@ -0,0 +1,12 @@
+import "./App.css";
+import Navbar from "./Components/Navbar/Navbar";
```

Comment:
Suggestion for future: You can create a `index.js` file inside Components and export all The components from that file, it will look much better.